scheduler/src/main/java/org/apache/mesos/elasticsearch/scheduler/state/StatePath.java

```diff
@@ -43,4 +43,9 @@ public Boolean exists(String key) throws IOException {
         }
         return exists;
     }
+
+    public void rm(String key) throws IOException {
+        key = key.replace(" ", "");
```

Comment:
Fair point. Not sure. Another method had it. I obviously had problems with spaces in the past. Will remove.